docs/pipeline-configuration-files/specifications.mdx

```diff
@@ -276,22 +278,31 @@ beginning.
   identify the processor inside the pipeline. If multiple processors inside the
   pipeline use the same ID, provisioning will fail.
 
-### type
+### plugin
 
 - **Data Type**: String
 - **Required**: Yes
 - **Default**: None
-- **Description**: Defines the processor type (e.g. `js`). Check out the
-  [processors documentation](/docs/processors/getting-started) to find out which
-  processors are supported.
+- **Description**: Defines the processor's plugin name (e.g. `field.set`). Check out the
+  [processors documentation](/docs/processors/builtin) to find the list of builtin processors
+   we provide.
+
+### condition
+
+- **Data Type**: String
+- **Required**: No
+- **Default**: `True` (all records will passed to the processor)
+- **Description**: a Go template expression that will be used as a condition to pass the record to the processor or not.
+the Go template expression will be evaluated, and the evaluated value should either be `True` or `False`, `True` means
+the record will be passed to the processor, `False` means it will be skipped.
```

Comment:
Have you been secretly programming in python⁉️ 🐍 I added some suggestions on clarifying what data us used when evaluating the expression and what happens with "skipped" records (skipped could let somebody think they get filtered out). I also added a link to Go templates docs, as not everybody knows about them.
```suggestion
- **Description**: A [Go template expression](https://pkg.go.dev/text/template) that will be used as a condition to pass the record to the processor or not.
The Go template expression will be evaluated using each record and needs to produce `true` or `false`. If it produces `true`
the record will be passed to the processor, `false` means it will continue down the pipeline without getting processed.
```

Reply:
super neat and clear suggestion! thanks